iceoryx_hoofs/README.md

```diff
@@ -1,68 +1,76 @@
 
 # Eclipse iceoryx hoofs overview
 
-The iceoryx hoofs (**H**andy **O**bjects **O**ptimised **F**or **S**afety) are our basic building blocks - the foundation of
-iceoryx. There are a wide variety of building blocks grouped together in categories
-or namespace, depending on where or how they are used.
+The iceoryx hoofs (**H**andy **O**bjects **O**ptimised **F**or **S**afety) are our basic
+building blocks - the foundation of iceoryx. There are a wide variety of building blocks
+grouped together in categories or namespace, depending on where or how they are used.
 
 ## Categories
 
 | Namespace       | Short Description |
 |:---------------:|:------------------|
 | [cxx](#cxx) | Since we are not allowed to use C++17 as well as the heap or exceptions we implemented constructs like `optional`, `expected` or `variant` so that we can be as modern as possible. Furthermore, you can find here constructs which are mentioned in the [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines) as well as STL re-implementations of container like `vector` which are relocatable an can be placed into the shared memory. |
-| [concurrent](#concurrent) | You should never use concurrent constructs like `mutex`, `semaphores`, `atomic` etc directly in our codebase. At the moment we still have some exceptions to this guideline but the idea is that all classes which are using them are stored under concurrent and have to undergo more tests then the usual non concurrent class. For instance we try to provide stress tests for them. In here you can find `fifo`, `smart_lock`, `sofi`, `trigger_queue` and much more. |
-| [design_pattern](#design-pattern) | Certain code patterns which are repeating themselves all over the code are abstracted and stored in here. At the moment we only have the creation pattern but maybe there will be more. |
-| [error-handling](#error-handling) | The Error-Handler to in iceoryx for handling fault cases. |
+| [concurrent](#concurrent) | You should never use concurrent constructs like `mutex`, `semaphores`, `atomic`, etc directly in our codebase. At the moment we still have some exceptions to this guideline but the idea is that all classes which are using them are stored under concurrent and have to undergo more tests then the usual non concurrent class. For instance we try to provide stress tests for them. This module provides classes like `fifo`, `smart_lock`, `sofi`, `trigger_queue` and much more. |
```

Comment:
```suggestion
| [concurrent](#concurrent) | You should never use concurrent constructs like `mutex`, `semaphores`, `atomic`, etc. directly in our codebase. At the moment we still have some exceptions to this guideline but the idea is that all classes which are using them are stored under concurrent and have to undergo more tests then the usual non concurrent class. For instance we try to provide stress tests for them. This module provides classes like `fifo`, `smart_lock`, `sofi`, `trigger_queue` and much more. |
```

Reply:
Oh boy, I need some vacation. While my eyes moved from your suggestion to the code I forgot about the period and added a semicolon before `etc`